contracts/finality/src/contract.rs

```diff
@@ -107,6 +109,36 @@ pub fn execute(
             // Validate and set the new admin address
             Ok(ADMIN.execute_update_admin(deps, info, Some(api.addr_validate(&admin)?))?)
         }
+        ExecuteMsg::PruneData {
+            rollup_height,
+            max_signatures_to_prune,
+            max_pub_rand_values_to_prune,
+        } => {
```

Comment:
Do you think we should follow the convention in other handlers, e.g.,

```suggestion
        ExecuteMsg::PruneData {
            rollup_height,
            max_signatures_to_prune,
            max_pub_rand_values_to_prune,
        } => handle_prune_data(...)
```